contrib/platform/src/com/sun/jna/platform/win32/COM/util/Convert.java

```diff
@@ -21,11 +23,33 @@
 import com.sun.jna.platform.win32.WTypes;
 import com.sun.jna.platform.win32.WinDef;
 import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WTypes.BSTR;
 
-public class Convert {
-	
+/**
+ * This class is considered internal to the package.
+ */
+class Convert {

```

Comment:
I'm wondering why these methods wouldn't simply be part of the `VARIANT` class definition.

OK if that's outside the scope of this PR, I realize this class was already there.